torchsde/_core/adjoint.py

```diff
@@ -167,24 +70,16 @@ def backward(ctx, *grad_outputs):
         dt_min = ctx.dt_min
         adjoint_options = ctx.adjoint_options
 
-        params = misc.make_seq_requires_grad(sde.parameters())
-        n_tensors, n_params = len(ans), len(params)
-
+        aug_state = (ys[-1], grad_ys[-1], *[torch.zeros_like(param) for param in params])
```

Comment:
Nitpick: I think this would be slightly neater as `(ys[-1], grad_ys[-1]) + tuple(torch.zeros_like(param) for param in params)`.

Reply:
Sure. I'll be using lists for the second part now, as list comprehension is slightly faster.

Reply:
Interesting, I didn't know listcomps were faster. Ty.